cluster/examples/kubernetes/ceph/upgrade-from-v1.1-crds.yaml

```diff
@@ -1,6 +1,3 @@
-# Latest custom resource definitions with schema validations
```

Comment:
Why removing explanation/instructions from the top of this?

Reply:
Oh, I meant to put this comment back.